### PR TITLE
add date input widgets

### PR DESCRIPTION
**Pull request by 2e2a**
Copy `euth_wagtail/contrib/widgets` to core

**Incomplete:** Javascript and dependencies are missing.

### CODE REVIEW

**Comment by MagdaN** [comment]
I fixed some stuff (basically using local times in browser but UTC in DE) for this in our release2.0 branch. I would suggest to use this https://github.com/liqd/euth_wagtail/commit/8e026bffccd920528a38df43afc075e46d3af5d1

Also newest version of Flatpickr should be used for proper Safari Support.

**Comment by 2e2a** [comment]
> I fixed some stuff (basically using local times in browser but UTC in BE) for this in our release2.0 branch. I would suggest to use this liqd/euth_wagtail@8e026bf
>
> Also newest version of Flatpickr should be used for proper Safari Support.

Thanks for the hint, will consider it, when we start using it.

File: adhocracy4/contrib/widgets.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    class Media:
+        js = (staticfiles_storage.url('flatpickr.min.js'),
+              staticfiles_storage.url('js/dateTimeInput.js'))
+        css = {'all': [staticfiles_storage.url('flatpickr.min.css')]}
+
+    input_type = 'text'
+    format_index = 0
+
+    # becomes a public value in Django 1.10
+    def _format_value(self, value):
```
Comment (slomo) [inline adhocracy4/contrib/widgets.py]: I am not sure if and why we needed to override this.

**Comment by 2e2a** [comment]
Closing this for now. Might reopen it when implementing the date picker.

**Pull request closed without merging**